pom.xml

```diff
@@ -88,11 +83,6 @@
             <artifactId>ezmorph</artifactId>
```

Comment:
This is in scope `compile` but we release plugin as a single jar without the dependencies and ReadyAPI does not provide it as well. Looks like something that cannot be used on production anyway. Shouldn't we get rid of this?

Reply:
Without this library plugin will not import collections throwing `java.lang.NoClassDefFoundError: net/sf/ezmorph/Morpher`
It's used for json processing:
`at net.sf.json.util.CycleDetectionStrategy.<clinit>(CycleDetectionStrategy.java:37) ~[?:?]
	at net.sf.json.JsonConfig.<clinit>(JsonConfig.java:61) ~[?:?]
	at net.sf.json.groovy.JsonSlurper.<init>(JsonSlurper.java:43) ~[?:?]
	at com.smartbear.ready.plugin.postman.utils.PostmanJsonUtil.seemsToBeJson(PostmanJsonUtil.java:26) ~[?:?]`